v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "    remote_base_run = AutoMLRun(experiment=experiment, run_id=BASE_RUN_ID)\n",
+    "    print(f\"Using previously trained model. Run ID: {BASE_RUN_ID}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Test set inference "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.1 Inference the best TCN model\n",
+    "\n",
+    "We create an output folder which will be used to save the output of our experiments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an output folder\n",
+    "OUTPUT_DIR = os.path.join(os.getcwd(), \"forecast_output\")\n",
+    "os.makedirs(OUTPUT_DIR, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.1.1 Get Inference Pipeline Environment\n",
+    "To trigger an inference pipeline run, we first need a running environment that contains all the appropriate packages for the model unpickling. This environment can be either assessed from the training run or using the `yml` file that comes with the model. In this section we use the environment from the training run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from scripts.register_model import get_best_automl_run\n",
+    "\n",
+    "best_run = get_best_automl_run(training_pipelin_run)\n",
+    "inference_env = best_run.get_environment()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.runconfig import RunConfiguration\n",
+    "\n",
+    "run_config = RunConfiguration()\n",
+    "run_config.environment = inference_env"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.1.2 Build and submit the inference pipeline\n",
+    "\n",
+    "The inference pipeline will create two different output formats, 1) a tabular dataset that contains the prediction and 2) an `OutputFileDatasetConfig` that can be used for the sequential pipeline steps. The `inference_script_tcn.py` performs a rolling evaluation on the test set with the step size of 24 hours. This generates the same results as if the model was put in production and generated a 24-hour forecast once a day using the most recent data context. This is an efficient way of testing historical model performance at a tiny fraction of the compute and time commitment costs. For a more detailed information on rolling evaluation see the this [document](https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-auto-train-forecast-v1#evaluating-model-accuracy-with-a-rolling-forecast) for a more detailed explanation.\n",
```

Comment:
In the next few sections, we are using variables with prefix backtest like backtest_tcn, backtest_base etc. Should we introduce the word backtest with prior to the usage? Maybe with this line "This generates the same results as if the model was put in production and generated a 24-hour forecast once a day using the most recent data context."?